testing/test_functions.py

```diff
@@ -52,7 +52,7 @@ def test_format_version(version, monkeypatch, scheme, expected):
 
 def test_dump_version_doesnt_bail_on_value_error(tmpdir):
     write_to = "VERSION"
-    version = VERSIONS['exact']
+    version = str(VERSIONS['exact'].tag)
```

Comment:
not sure what isthis for - would this silence possible unit test failures that could detect the fixed bug?